Yosemite/Yosemite/Model/Storage/ShippingLabel+ReadOnlyConvertible.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+import Storage
+
+// Storage.ShippingLabel: ReadOnlyConvertible Conformance.
+//
+extension Storage.ShippingLabel: ReadOnlyConvertible {
+    /// Updates the Storage.ShippingLabel with the a ReadOnly ShippingLabel.
+    ///
+    public func update(with shippingLabel: Yosemite.ShippingLabel) {
+        self.siteID = shippingLabel.siteID
+        self.orderID = shippingLabel.orderID
+        self.shippingLabelID = shippingLabel.shippingLabelID
+        self.carrierID = shippingLabel.carrierID
+        self.dateCreated = shippingLabel.dateCreated
+        self.packageName = shippingLabel.packageName
+        self.rate = shippingLabel.rate
+        self.currency = shippingLabel.currency
+        self.trackingNumber = shippingLabel.trackingNumber
+        self.serviceName = shippingLabel.serviceName
+        self.refundableAmount = shippingLabel.refundableAmount
+        self.status = shippingLabel.status.rawValue
+        self.productIDs = shippingLabel.productIDs
+        self.productNames = shippingLabel.productNames
```

Comment:
The `self` keywords are not needed. 😊 

```suggestion
        siteID = shippingLabel.siteID
        orderID = shippingLabel.orderID
        shippingLabelID = shippingLabel.shippingLabelID
        carrierID = shippingLabel.carrierID
        dateCreated = shippingLabel.dateCreated
        packageName = shippingLabel.packageName
        rate = shippingLabel.rate
        currency = shippingLabel.currency
        trackingNumber = shippingLabel.trackingNumber
        serviceName = shippingLabel.serviceName
        refundableAmount = shippingLabel.refundableAmount
        status = shippingLabel.status.rawValue
        productIDs = shippingLabel.productIDs
        productNames = shippingLabel.productNames
```

Reply:
Removed here: https://github.com/woocommerce/woocommerce-ios/pull/3176/commits/344ec72aaff55eb55026eacd893be18dbc55aa2d